src/model/footsteps/hooks/index.ts

```diff
@@ -9,11 +9,15 @@ const loadFootstepsAudio = () => {
   return footsteps;
 };
 
+//オーディオインスタンス
+const footstepsAudio = loadFootstepsAudio();
+
 const playAudio = (audio: HTMLAudioElement) => {
+  if (!audio.paused) {
+    audio.pause();
+    audio.currentTime = 0; //タイム0
```

Comment:
👍 

nits:
plz add a space before the comments

imho:
I prefer to have line breaks in comments, so it's a bit of an issue for me, but either way is fine.